ykrt/pass_finder/try_passes.py

```diff
@@ -176,6 +181,10 @@ def main(logf, is_prelink):
     binary_split(logf, passes, is_prelink)
 
 if __name__ == "__main__":
+    if '--doc' in sys.argv:
```

Comment:
`--help` is more conventional

Reply:
addressed here: 934d4f63bef0ffc771a4a25a8ccef762f0ec3d28